include/variables.h

```diff
@@ -192,7 +193,31 @@ extern s32 gPauseMenuTargetPosY;
 extern s32 gPauseMenuCursorTargetOpacity;
 extern u8 gPauseMenuTextScrollInterpEasingLUT[16];
 extern u8 gPauseMenuPageScrollInterpEasingLUT[16];
-
+extern s32 D_80270700[1];
+extern f32 D_80270704;
+extern f32 D_80270708;
+extern s32 mapMarioXPos; //mario xPos on map tab
+extern s32 mapMarioYPos; //mario xPos on map tab
+extern f32 mapCursorXPos; //cursor X pos float map tab
+extern f32 mapCursorYPos; //cursor Y pos float map tab
+extern s32 mapCursorCurrentOption;
+extern s32 mapCursorCurrentOptionCopy;
+extern s32 D_80270724;
+extern f64 D_80270080;
+extern f64 D_80270088;
+extern f64 D_80270090;
+extern f64 D_80270098;
+extern f64 D_802700A0;
+extern f64 D_802700A8;
+extern f64 D_802700B0;
+extern f64 D_802700B8;
+
+
+extern s32 D_8024FA30[1];
+
+extern Pause80250668* D_80250668[6];
+extern PauseMapSpaces* D_802502B8[5];
```

Comment:
These two variables should be just arrays, not need for the pointers. This will let you remove the casts on D_802502B8.

Additionally, D_80250668 is a fake symbol. It's actually the end element of an array, so this address will need changed.

Reply:
I also suspect D_802502B8 has a different length, as is some accesses to this are UB.